lib/repo_tools.pm

```diff
@@ -258,9 +258,6 @@ sub rmt_wizard {
         record_soft_failure 'bsc#1195759';
         zypper_call 'in rmt-server';
     }
-    else {
```

Comment:
I think you can't remove it directly. This workaround is for rmt case without DEV_PATH, for DEV_PATH you can zypper in rmt_server instead. So there should be two branches.

Reply:
I would not remove the else because then other failure than `scriptlet fail` will pass.
But I don't know how to suppress the notification. I hope the bug is fixed soon and the workaround can be removed.

Reply:
In fact, I think there are two status for different zypper version, on DEV_PATH there is no bug found, without DEV_PATH, it will fail with return code of 107 and waning message in log.
So why not use the return value of zypper_call as the condition of the workaround such as 
if ($ret == 107)
    softfailure

Reply:
How about add extra elif (get_var("DEV_PATH")) instead of remove the else? If i understand correctly.

Reply:
Return code 107 could be used.

Reply:
Using return code 107, please help to review, thanks.

Reply:
Please run VRs with latest code.